ktlint-ruleset-template/build.gradle.kts

```diff
@@ -0,0 +1,71 @@
+plugins {
+    `ktlint-kotlin-common` // replace it with 'org.jetbrains.kotlin.jvm'
+    `java-library`
+    `maven-publish`
+}
+
+group = "com.github.username"
+
+val sourcesJar by tasks.registering(Jar::class) {
+    dependsOn(tasks.classes)
+    archiveClassifier.set("sources")
+    from(sourceSets.main.get().allSource)
+}
+
+val javadocJar by tasks.registering(Jar::class) {
+    dependsOn(tasks.javadoc)
+    archiveClassifier.set("javadoc")
+    from(tasks.javadoc.get().destinationDir)
+}
+
+artifacts {
+    archives(sourcesJar)
+    archives(javadocJar)
+}
+
+val ktlint: Configuration = configurations.create("ktlint")
+
+dependencies {
+    ktlint(projects.ktlint)
+
+    compileOnly(projects.ktlintCore)
+
+    testImplementation(projects.ktlintCore)
+    testImplementation(projects.ktlintTest)
+    testImplementation(libs.junit5)
+    testImplementation(libs.assertj)
+}
+
+tasks.test {
+    useJUnitPlatform()
+}
+
+tasks.register<JavaExec>("ktlint") {
```

Comment:
The `ktlint` task's configuration in `ktlint-ruleset-template` is not same with `ktlint` task in root `build.gradle`, I'm not sure which one is right.

https://github.com/pinterest/ktlint/blob/c42647182196339f08ff14cea9dd0321d69014fc/build.gradle#L30-L39